src/components/starboard.ts

```diff
@@ -341,13 +341,32 @@ export default class Starboard extends BotComponent {
         }
     }
 
+    // Check if the # of reactions is full, and there is no negative emojis reacted yet
+    should_delete_reaction(reaction: Discord.MessageReaction) {
+        return (
+            reaction.message.channel === this.wheatley.channels.memes &&
```

Comment:
I'm not sure if the channel object for the message is guaranteed to be the same channel object fetched earlier. I'd recommend comparing the channel ids just to be safe.